phpstan-baseline.neon

```diff
@@ -0,0 +1,110 @@
+
+
+parameters:
+	ignoreErrors:
```

Comment:
@rvanlaak I think it would be better we fixed these errors than ignoring them.

Reply:
Absolutely, but it could cause bugs where this PR does focus on adding static analyses to begin with. That way new code will be covered. What about fixing it in a new PR?